src/System.Text.Encodings.Web/src/System.Text.Encodings.Web.csproj

```diff
@@ -2,9 +2,10 @@
   <PropertyGroup>
     <RootNamespace>System.Text.Encodings.Web</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Configurations>netstandard-Debug;netstandard-Release;netstandard2.1-Debug;netstandard2.1-Release</Configurations>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release;netstandard2.1-Debug;netstandard2.1-Release</Configurations>
```

Comment:
Adding a new netcoreapp config to the source implementation (to make use of intrinsics which are only available on core).

cc @ericstj

Reply:
So adding a config here is OK, so long as its in packages.  One thing you need to be careful about is adding a configuration where an app might rollforward with a newer package and downgrade the inbox implementation with lesser one.

Consider: App targets netcoreapp3.0, references a new package, say from 5.0 wave.  App rolls forward to 3.1.  5.0 binary might have higher version than 3.1 inbox, but be missing your feature.

Reply:
Addressing this in https://github.com/dotnet/corefx/pull/42069